README.md

```diff
@@ -1,8 +1,74 @@
+Springboot embedded `Tomcat` server serves as a web servlet API implementation to handle `HTTP` requests. `Repository` pattern is used as a data layer and `IEX API` is used as a source of market data `PostMan` and `Swagger` have been used extensively to
+test the app end-points. The decoupling of data and app make it fault-tolerant.
+- **[Cloud & DevOps](./cloud_devops)**: Not started
+- **[Hadoop](./hadoop)**: Not started
+- **[Spark/Scala](./spark)**:  Not started
+
+## Professional Experiences
+
+**Software Developer,  Jarvis, Toronto (2020-Present):** Primary Responsibility was to develop `Java`, `BASH/SQL`, `Springboot` based services and architectures utilizing `GitFlow` and other industry-recognized tools and best practices,  and collaborating with the team and other team members maintaining standard agile practices. Working in this role made me a better team player and gave me invaluable work environment experience to solve problems individually with minimal supervision.
+
+**Research Assistant, Western University, London, ON (February, 2020-April, 2020):** Primary responsibility was to develop a framework to integrate Quantum computing with Computer networking. Worked with `QCL` and `Python` to perform experiments. Finding scopes to utilize newer methods to existing problems improved my problem-solving ability.
+
+**Research Assistant, Western University, London, ON (September 2019 - January 2020):** Mentored the current lab students and collaborated with labs to deliver an automation software solution. It was a medical imaging software that provides an efficient way to track and count blood cells from dyed brain images of rats. The solution was adaptable with different dyes and several image scenarios and distractions. The solution was written in `C++` and `iopng` library was used to read and write images to reduce latency. However, the solution was packaged as a `Mex` file to be invoked from `Matlab` for user convenience. This experience grew leadership and increased my collaboration abilities with clients.
+
+**Research Assistant, Western University, London, ON (September 2017 - August 2019):** Designed and implemented my proposed algorithm to improve image processing (denoising) performance which was my master's research and helped me to achieve the degree. `BM3D` is the state-of-the-art algorithm for image denoising. The proposed algorithm works on top of `BM3D` and improves the overall denoising performance. The proposed algorithm beat the performance of `BM3D` significantly in all noise levels and varieties of images. The codebase was written in `C++`. To reduce the time complexity approaches like `Dynamic programming` was used while designing and low latency `C` libraries like `iopng` were used to read and write the image. Designing and implementing algorithms with minimal supervision made me confident in working individually.
```

Comment:
which was my master's research topic.
Last line: made me confident to work individually